### ISSUE TITLE
4.1j appears to be a rewording of supermajority but is confusing

### ISSUE THREAD

**Issue by dcbullock** [SMPTE/standards-operations-manual#1]
@mrmxf 

Clarify 4.1j so it is distinct from 4.3.2 or remove

**Comment by dcbullock** [comment]
v3.1 for reference:

4.1.j.

For decisions on significant questions affecting the content of Engineering Documents or motions made in any Group, Consensus shall be obtained. Consensus shall be determined to exist either when there are no objections or when no more than 1/3 of the Participant Members who respond in favor or opposed are opposed. All determinations of Consensus shall be recorded in the Minutes. The use of "straw polls" and other means of achieving Consensus are encouraged.


4.3.2 Consensus Vote

A Consensus Vote is a vote requiring a defined supermajority for approval. The supermajority is defined as follows: Of the Participant Members who vote either Yes or
Version 3.1: Approved 9 February 2015 17
No, there must be at least twice as many Yes votes as No votes. Abstentions shall not be factored into the calculation, nor shall votes not cast.

**Comment by dcbullock** [comment]
I think this needs discussion.  I am currently interpreting these two sections such that 4.3.2 is a subset of 4.1.j - a Consensus Vote is a type of "significant question" and as such is required to conform to 4.1.j, which it does.  4.3.2 then goes on to add further constraints by specifying that a Consensus Vote requires a super-majority, which in turn is then defined.  The super-majority definition in 4.3.2 includes the minimum votes requirement and the consensus in 4.1.j only includes the 1/3 ratio requirement.

Is there an error in 4.1.j in that "significant questions" should also include the minimum vote requirement or is it intentional to leave it out?

**Comment by dcbullock** [comment]
I think 4.3.2 is fine as is and only applies to Votes, which are only taken at the TC level.

4.1.j defines Consensus as applied in a meeting of any Group and is inconsistent because it assumes all Groups are due-process groups.  Only the TCs (and ST?) are due process.  Sub-groups also explicitly give all members equal status - P-membership in the parent TC has no meaning in sub-group happenings.

The sub-group section of the OM defines how they should operate including requiring consensus to make recommendations to the TC.

I suggest:


4.1.j
Where otherwise not explicitly indicated in this document, including but not limited to Votes, Written Ballots, and other due process procedures, decisions made by any Group shall be made by Consensus.  Consensus shall be determined to exist either when there are no objections or when no more than 1/3 of the members who respond in favor or opposed are opposed. All determinations of Consensus shall be recorded in the Minutes. The use of "straw polls" and other means of achieving Consensus are encouraged.

**Comment by mrmxf** [comment]
I like the spirit of the fix, but not the implmenetation which is basically "if you can't find a loophole elsewhere in the document then...."

**_TL;DR;_** It would be nice if the text was more of the form "in this condition do this else that"

**Observations**

1. The initial comment arose becuase the OM is often cherry-picked when there is confusion. I have found that  "intentional wiggle room" from two different sections are used by proponents with conflicting views. On the spur of the moment, it's hard to arbitrate without significan knowledge of the rest of the document.
2. The phrase due process is often misused - §6.2.1.j implies that everything with documented decisions is due process whereas the OMs sub-divide into what I think of as **formal-process** i.e. ST, TC, BoG where Roberts Rules, voice-votes and strict "address the chair" apply and "relaxed process" for subgroups where a more relaxed culture exists
3. Adding lots of new names for things can be bad.
4. Having precise meanings for terms encourages their use in precise situations.

I would be tempted to define _formal process_ and _relaxed process_ and look at the impact on the document. I suspect it would remove some of the more tortuous text.

My rewrite of 4.1.j to address the "being read in a panic" use case would be something like this:

Consensus shall be obtained for decisions on significant questions affecting the content of Engineering Documents or motions made in any Group,

Consensus exists when the documented minutes show that there were 2 or more approvial votes for every opposing vote.

Methods to achieve consesnus that include calls for "any objections", "any supporters" and "straw polls" that quickly establish consensus appropriate to the formality of the group are encouraged.
